include/openenclave/internal/trace.h

```diff
@@ -28,6 +28,17 @@ extern oe_log_level_t _log_level;
 #define OE_MAX_FILENAME_LEN 256U
 
 #if !defined(OE_BUILD_ENCLAVE)
+typedef void (*oe_log_callback)(
+    void* context,
+    bool is_enclave,
+    const char* time,
+    long int usecs,
+    oe_log_level_t level,
+    const char* message);
+oe_result_t oe_set_log_callback(void* context, oe_log_callback callback);
+extern void* _oe_log_context;
```

Comment:
what is the intention of making this `extern`?

Reply:
Yes definition in traceh.c and declaration in trace.h can make these variables global and avoid potential redefinition error.